modules/jetson-benchmarks.nix

```diff
@@ -0,0 +1,30 @@
+{ pkgs
+, config
+, lib
+, ...
+}:
+with lib;
+let
+  cfg = config.jetson-benchmarks;
+
+  benchmarkPackage = pkgs.callPackage ../jetson-benchmarks/default.nix {
+    targetSom = config.hardware.nvidia-jetpack.som;
+    cudaPackages = pkgs.nvidia-jetpack.cudaPackages;
+  };
+in
+{
+  options.jetson-benchmarks = {
```

Comment:
This module isn't necessary. It doesn't modify the system other than putting the benchmark package into environment.systemPackages and that can be done by the user if they want

Reply:
Sorry about that, forgot to remove it